Create CONTRIBUTING.md (#92)
CrowdStrike/falconpy: CONTRIBUTING.md

# Contributing to this repository <!-- omit in toc -->

## Getting started <!-- omit in toc -->
_Welcome!_ We're excited you want to take part in the FalconPy community! 

> Please review this document for details regarding project needs, contributor requirements and our pull request process. If you have any questions, please let us know by

Reviewer: ```suggestion
Please review this document for details regarding getting started with your first contribution, packages you'll need to install as a developer, and our pull request process. If you have any questions, please let us know by
```

posting in your question in the [discussion board](https://github.com/CrowdStrike/falconpy/discussions).

### Before you begin

> - Have you read the [Code of Conduct](CODE_OF_CONDUCT.md)?

Reviewer: ```suggestion
- Have you read the [Code of Conduct](CODE_OF_CONDUCT.md)? The Code of Conduct helps us establish community norms and how they'll be enforced.
```

### Table of Contents
- [How you can contribute](#how-you-can-contribute)
- [Pull Requests](#pull-requests)
- [Suggestions](#suggestions)

## How you can contribute

> - Submit a [bug report](https://github.com/CrowdStrike/falconpy/issues).

Reviewer: ```suggestion
- See something? Say something! Submit a [bug report](https://github.com/CrowdStrike/falconpy/issues) to let the community know what you've experienced or found. Bonus points if you suggest possible fixes or what you feel may resolve the issue. For example: "_Attempted to use the XZY API class but it errored out. Could a more descriptive error code be returned?_"
```

We use GitHub issues to track bugs. Report a bug by opening a [new issue](https://github.com/CrowdStrike/falconpy/issues).

### GitHub's discussion board is used for questions, suggestions and feedback.
We use GitHub's discussion board functionality to handling community discussions related to 
[questions](https://github.com/CrowdStrike/falconpy/discussions/categories/q-a), 
[feedback](https://github.com/CrowdStrike/falconpy/discussions/categories/general) 
or [functionality enhancements](https://github.com/CrowdStrike/falconpy/discussions/categories/ideas).

## Pull Requests

### All contributions will be submitted under the Unlicense license
When you submit code changes, your submissions are understood to be under the same Unlicense [license](LICENSE) that covers the project. 
If this is a concern, contact the maintainers before contributing.

### Additional contributor package requirements
The file `requirements-dev.txt` contains additional Python requirements used for developing unit tests and for accessing the integrated debugger within FalconShell.
Review this file's contents and install these requirements before proceeding:
+ flake8
+ pytest
+ pytest-cov
+ coverage

> + iPython

Author: Not really, just being over-verbose. :smile:
